scarlet/source.py

```diff
@@ -597,3 +597,114 @@ def make_initial(self, img, bg_rms, thresh=1., symmetric=True, monotonic=True, c
             # keep the peak sed
             logger.INFO("Using peak SED for source at {0}/{1}".format(self.center_int[0], self.center_int[1]))
         return sed.reshape((1,B)), morph.reshape((1, morph.shape[0], morph.shape[1]))
+
+class PSFDiffKernel(Source):
```

Comment:
If you derive this from `ExtendedSource`, you can use all of its `make_initial`.

Reply:
The only thing needed as a SED that is set to `(0,0,1,0,0)` or the like, but you can use the morphology from `ExtendedSource`

Reply:
It isn't quite that simple, because we also don't create the detection coadd. So I created an `_init_morph` method of `ExtendedSource` that only initializes the morphology, and call that from `PSFDiffKernel._make_initial`.